src/common/dav/ulog.h

```diff
@@ -74,12 +75,33 @@ VEC(ulog_next, struct ulog *);
 typedef uint64_t ulog_operation_type;
 
 #define ULOG_OPERATION_SET		(0b000ULL << 61ULL)
+#ifdef	WAL_SUPPORTS_AND_OR_OPS
 #define ULOG_OPERATION_AND		(0b001ULL << 61ULL)
 #define ULOG_OPERATION_OR		(0b010ULL << 61ULL)
+#else
+#define ULOG_OPERATION_CLR_BITS		(0b001ULL << 61ULL)
+#define ULOG_OPERATION_SET_BITS		(0b010ULL << 61ULL)
+#endif
 #define ULOG_OPERATION_BUF_SET		(0b101ULL << 61ULL)
 #define ULOG_OPERATION_BUF_CPY		(0b110ULL << 61ULL)
```

Comment:
```suggestion
#define ULOG_OPERATION_SET		(0x0 << 61ULL)
#ifdef	WAL_SUPPORTS_AND_OR_OPS
#define ULOG_OPERATION_AND		(0x1 << 61ULL)
#define ULOG_OPERATION_OR		(0x2 << 61ULL)
#else
#define ULOG_OPERATION_CLR_BITS		(0x1 << 61ULL)
#define ULOG_OPERATION_SET_BITS		(0x2 << 61ULL)
#endif
#define ULOG_OPERATION_BUF_SET		(0x5 << 61ULL)
#define ULOG_OPERATION_BUF_CPY		(0x6 << 61ULL)
```

Reply:
Please make an exemption for gcc v4.3+ binary constant extension.